modules/host/host.go

```diff
@@ -245,13 +235,15 @@ func newHost(dependencies dependencies, cs modules.ConsensusSet, tpool modules.T
 		persistDir: persistDir,
 	}
 
+	// Call stop in the event of a partial startup.
```

Comment:
perhaps in another PR. I'm not confident about the best way to make sure that startup is clean.